worker/firewaller/firewaller.go

```diff
@@ -1194,10 +1194,23 @@ func (fw *Firewaller) relationLifeChanged(tag names.RelationTag) error {
 	}
 	rel := results[0].Result
 
-	dead := notfound || rel.Life == params.Dead
+	dead := notfound || rel.Life == params.Dead || rel.Suspended
```

Comment:
please rename to something more appropriate, it no longer means "dead" (removed implies it *was* dead, so that one was OK)

alternatively just check `known && (dead || rel.Suspended)`. you check rel.Suspended inside anyway

Reply:
Renamed to "gone". We use this further down as well.